server/src/routes/api/v1/categoriesPastasRouter.js

```diff
@@ -0,0 +1,26 @@
+import express from "express"
+import objection from 'objection'
+import { Pasta } from "../../../models/index.js"
+const { ValidationError } = objection
+import cleanUserInput from "../../../services/cleanUserInput.js"
+
+const categoriesPastasRouter = new express.Router({ mergeParams: true })
+
+categoriesPastasRouter.post("/", async (req, res) => {
+  const { body } = req
+  const formInput = cleanUserInput(body)
+  const { name, description } = formInput
+  const { categoryId } = req.params 
+  
+  try {
+    const newPasta = await Pasta.query().insertAndFetch({ name, description, categoryId })
+    return res.status(200).json({ pasta: newPasta })
```

Comment:
successful POST request should respond with a status of 201
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201